cmd/virt-api/virt-api.go

```diff
@@ -136,6 +137,12 @@ func (app *virtAPIApp) Run() {
 		},
 	}
 	swagger.InstallSwaggerService(config)
+	openapiConf := restfulspec.Config{
+		WebServices:    restful.RegisteredWebServices(),
```

Comment:
Now we can't register all webservices anymore. If we do it this way, it will start documenting the swagger docs service too, and the other way around:

```
  '/swaggerapi/{a}/{b}':
    get:
      produces:
        - application/json
      operationId: getDeclarations
      responses:
        '200':
          description: OK
```

Changing it to `[]*WebService{ws}` for the openapi and the swagger config should do the trick.